packages/legend-application-repl/src/stores/LegendREPLDataCubeEngine.ts

```diff
@@ -191,10 +191,12 @@ export class LegendREPLDataCubeEngine extends DataCubeEngine {
     source: DataCubeSource,
     options?: DataCubeExecutionOptions | undefined,
   ) {
+    const startTime = performance.now();
```

Comment:
pretty cool! I learnt something new, we can start using this in the `DataCubeTaskService` as well